ofcourse/yamls/site.yaml

```diff
@@ -7,6 +7,8 @@ course:
     times: Tues & Thurs 6:30PM - 7:45PM
     course: 4085.582.01
     location: Orange Hall (ORN)-1380
+    repo: https://github.com/decause/hfoss
+    prod: https://hfoss-ritigm.rhcloud.com
```

Comment:
IIRC getting that info in OpenShift is actually a wsgi standard var, and we could just do it when the key isn't otherwise set. 

Also I really don't like the key "prod". Believe it or not, most professors won't immediately grasp that abbreviation. Something like "deploy_url" or "public_url" would be clearer.